DFE-Digital/Dfe.Academies.External
# fixing styles & adding missing text as per ticket

Testing fixes as per comments on below ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/103421?src=WorkItemMention&src-action=artifact_link

## Type of change

## Expected behaviour
a) Page font will be fixed for ' Schools must consult any stakeholders relevant to the conversion'
b) Missing 'Has the governing body consulted the relevant stakeholders?' text added

## Steps to reproduce issue (if relevant)
1. Browse to existing page and 'Has the governing body consulted the relevant stakeholders?' text will be missing
2. Browse to existing page and ' Schools must consult any stakeholders relevant to the conversion' text has no styling

## Steps to test this PR
1.  Browse to following page to ensure 2 issues are fixed:-
https://localhost:44350/school/application-school-consultation?appId=2147483647&urn=101934

## Prerequisites
n/a
